sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_greengrass_example.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.                "curl -s https://d2s8p88vqu9w66.cloudfront.net/releases/greengrass-nucleus-latest.zip &gt; greengrass-nucleus-latest.zip &amp;&amp; unzip greengrass-nucleus-latest.zip -d GreengrassCore",
</pre><p>Is this an official AWS release? Can you provide a link to some verification or documentation that this package is endorsed by AWS to download and install?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Yes, this is the official repository for the image :-( . I added a link to the docs</p>